Mausritter/Mausritter.html

```diff
@@ -0,0 +1,286 @@
+        <div class="sheet-top-line sheet-flex-between sheet-encumbrance-total">
+          <span class="sheet-equipment-name">Total</span>
+          <input type="number" name="attr_encumbrance_total" class="sheet-encumbrance" value="0" readonly="">
+        </div>
+    </div>
+</div>
+
+ <div class="sheet-thick-border">
+        <h3>Experience</h3>
+        <div class="sheet-flex">
+		    <div class="sheet-exp">
+                <h2>Pips</h2>
+                <input type="number" class="sheet-experiance-value" name="attr_Pips" value="0"> 
+            </div>
+            <div class="sheet-exp">
+                <h2>XP</h2>
+                <input type="number" class="sheet-experiance-value" name="attr_XP" value="0"> 
+            </div>
+            <div class="sheet-exp">
+                <h2>Level</h2>
+                <input type="number" class="sheet-experiance-value" name="attr_level" value="1"> 
+            </div>
+			<div class="sheet-exp">
+                <h2>Grit</h2>
+                <input type="number" class="sheet-experiance-value" name="attr_Grit" value="0">
+            </div>
+        </div>
+    </div>
+
+<div class="sheet-flex-between">
+    <div class="sheet-col-left sheet-thick-border">
+        <h3>Notes</h3>
+        <div class="sheet-flex">
+            <textarea name="attr_notes" style="width:97%;height:180px"></textarea>
+        </div>
+    </div>
+    <div class="sheet-col-right sheet-thick-border">
+        <h3>Notes 2</h3>
+        <div class="sheet-flex">
+            <textarea name="attr_notes2" style="width:97%;height:180px"></textarea>
+        </div>
+    </div>
+</div>
+</div>
+
+
+<!-- Sheet workers -->
+<script type="text/worker">
+// Equipment counter
+on('change:repeating_equipment remove:repeating_equipment', () => {
+  "use strict";
+  getSectionIDs('repeating_equipment', idArray => {
+    getAttrs(idArray.map(id => `repeating_equipment_${id}_encumbrance`), v => {
+      const total = Object.values(v).reduce((m, w) => (m + (parseFloat(w) || 0)), 0);
+      setAttrs({
+        encumbrance_total: total
+      });
+    });
+  });
+});
+</script>
+
+<!--
+&{template:ito-save} {{name=Test}} {{success=[[50]]}} {{roll1=[[1d100]]}}
+-->
+
+<rolltemplate class="sheet-rolltemplate-ito-save">
+    <table>
+        <caption>{{name}}</caption>
+        <tr>
+            <td class="template_label">Target:</td>
+            <td class="template_value">{{success}}</td>
+        </tr>
+        <tr>
+            <td class="template_label">Rolled:</td>
+            <td class="template_value">{{roll1}}</td>
+        </tr>
+        <tr style="background:#DFF0D8;">
+            {{#rollBetween() roll1 1 success}}
+			  <td class="template_value" colspan="2">Success</td>
+			{{/rollBetween() roll1 1 success}}
+		</tr>
+		<tr style="background:#F2DEDE;">
+			{{#rollGreater() roll1 success}}
+			  <td class="template_value" colspan="2">Failure</td>
+			{{/rollGreater() roll1 success}}
+        </tr>
+    </table>
```

Comment:
`<table>` tags are not allowed to be used for layouts in new sheets. This will need to be changed to use flex or grid layout.